install/kubernetes/helm/istio/values.yaml

```diff
@@ -425,3 +425,20 @@ tracing:
     # requests:
     #  cpu: 100m
     #  memory: 128Mi
+
+kiali:
+  enabled: false
+  replicaCount: 1
+  image:
+    repository: kiali/kiali
+    tag: 0.3.0.Alpha
+  resources: {}
+    # limits:
+    #  cpu: 100m
+    #  memory: 128Mi
+    # requests:
+    #  cpu: 100m
+    #  memory: 128Mi
+  dashboard:
+    username: admin
```

Comment:
That sounds very scarry

Reply:
is that the default user name/pwd?  Can this be configured via helm chart??

Reply:
This needs to be fixed in kiali https://github.com/kiali/kiali/blob/master/config.yaml#L5-L8 , for now, I think we can keep current logic and remove it once kiali fixed this.

Reply:
@aljesusg ^^ could be relevant

Reply:
There is an issue opened at here https://github.com/kiali/kiali/issues/201#issuecomment-393053069

Reply:
Ok I have it @abonas Thanks